github-cli.md

```diff
@@ -0,0 +1,110 @@
+**Quick Tip: Github and Command Line Interface (CLI)**
+======
+
+## **Overview**
+This is a quick guide to help you learn the basics about Github and CLI! If you have any more questions, or want other resources, [click here](https://www.atlassian.com/git/tutorials) for a good reference regarding Github and [ here](https://tutorial.djangogirls.org/en/intro_to_command_line/) for a good reference regarding CLI.
+
+## **Command Line Interface**
+<img src="https://memegenerator.net/img/instances/65841187/turn-off-the-gui-luke-use-the-command-line.jpg" width="300px" height="300px"/>
+
+### Introduction
+**What: ** CLI is a text-based interface which can be accessed through terminal (for OS X and Linux) or command prompt (for Windows). It allows a user to interact with the operating system seamlessly.
+
+**Why: ** While the Graphical User Interface (GUI) allows for editing and accessing files and folders, the CLI gives the user more power and precision.
+
+**NOTE: ** Some commands  are different for OS X and Linux vs Windows. Make sure you're using the right ones!
+
+### Commands
+
+#### Current Directory
+
+**OS X and Linux: ** `pwd` stands for "print working directory"
+
+**Windows: ** `cd` stands for "change directory" and can be used to switch to a new directory as well
+
+#### Learn More About a Command
+**OS X and Linux: ** `man <command>`
+
+**Windows: ** `<command> /?`
+
+#### List Files and Directories
+**OS X and Linux: ** `ls`
+
+**Windows: ** `dir`
+
+#### Change Current Directory
+**OS X, Linux, and Windows: ** `cd <path>`
+
+#### Move to the Parent Directory
+**OS X, Linux, and Windows: ** `cd ..`
+
+#### Create Directory
+**OS X, Linux, and Windows: ** `mkdir <name of directory>`
+
+### Cheat Sheet
+| Command | OS X and Linux | Windows |
+| :-----: | :------------: | :-----: |
+| Current Directory | `pwd` |  `cd`  |
+| Learn More | `man <command>` | `<command> /?` |
+| List Files and Directories | `ls` | `dir` |
+| Change Current Directory | `cd <path>` | `cd <path` |
+| Move to Parent Directory | `cd ..` | `cd ..` |
+| Create Directory | `mkdir <name of directory`> | `mkdir <name of directory>` |
+
+
+## **Github**
+<img src="https://memegenerator.net/img/instances/41740049/regular-peopleadd-me-on-facebook-me-fork-me-on-github.jpg" width="300px" height="300px"/>
+
+### Introduction
+**What: ** Git is a version control system which allows multiple people to work on a project at the same time (kind of like Google Docs).
+
+**Why: ** Imagine cooking with your friends: you all have to keep track of what the others are adding to the dish while making sure that what you add doesn't ruin it. Things get more complicated as more and more people join. The same is true of technical projects.
+
+**How: ** [Github!](https://github.com/)
```

Comment:
Worth explaining briefly that Github is the primary host of git projects, but as a concept git exists by itself as well